sdk/storage/storage-internal-avro/package.json

```diff
@@ -49,7 +49,7 @@
     "format": "dev-tool run vendored prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
     "integration-test": "npm run integration-test:node && npm run integration-test:browser",
     "integration-test:browser": "karma start --single-run",
-    "integration-test:node": "dev-tool run test:node-js-input --no-test-proxy -- -t 300000  \"dist-esm/test/*.spec.js\" \"dist-esm/test/node/*.spec.js\"",
+    "integration-test:node": "dev-tool run test:node-js-input --no-test-proxy -- --exclude 'test/snippets.spec.ts' -t 300000  \"dist-esm/test/*.spec.js\" \"dist-esm/test/node/*.spec.js\"",
```

Comment:
Seams this would cause test fail.
storage-internal-avro is not a project, it's just a folder to place some codes for storage-blob-changefeed, we should not test it as a project.